### PR TITLE
webauthn-2 use markup in its <h1> element

### PR DESCRIPTION
**Pull request by plehegar**
[[
Content of title and h1 do not match.
The document's title must be in the title element and in an h1 element.
]]
from [pubrules](https://www.w3.org/pubrules/?url=https%3A%2F%2Fwww.w3.org%2FTR%2F2019%2FWD-webauthn-2-20191126%2F&profile=WD&validation=simple-validation&noRecTrack=false&informativeOnly=false&echidnaReady=false&patentPolicy=pp2004)

cc @akshayku @wseltzer

### CODE REVIEW

**Comment by deniak** [comment]
For that specific use case, I feel like it would be better to sanitize the title before checking it. I think it's ok to remove the line breaks while fetching the title. If you are ok with my suggestion, I'll submit a PR.

**Comment by deniak** [comment]
Looking closely, removing the line break is not enough since we have the punctuation.

File: lib/exceptions.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
       "rule": "headers.copyright"
     }
   ],
+  "^webauthn-2$": [
+    {
+      "rule": "metadata.title"
```
Comment (deniak) [inline lib/exceptions.json]:
```suggestion
      "rule": "headers.h1-title"
```

**Comment by plehegar** [comment]
Added the change. Thank you Denis.

**Comment by deniak** [comment]
Merged and deployed

**Pull request merged**